### PR TITLE
Bump upstream base version to 0.111.0

### PR DESCRIPTION
**Pull request by srikanthccv**

### CODE REVIEW

File: components/components.go [hunk 1]
```diff
@@ -72,29 +95,52 @@ import (
...
 	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/sqlserverreceiver"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/sshcheckreceiver"
 	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/statsdreceiver"
 	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/syslogreceiver"
 	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/tcplogreceiver"
 	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/udplogreceiver"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/vcenterreceiver"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/wavefrontreceiver"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/webhookeventreceiver"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/windowseventlogreceiver"
```
Comment (nityanandagohain) [inline components/components.go:142]: is it necessary to add all of this in this receiver ?

Comment (srikanthccv) [reply components/components.go:142]: The unwritten policy we follow is: to include all the receiver/processor components so users can receive/process data from anywhere in the regular small self-hosted installations. But when it comes to exporters we only allow specific exporters.

Comment (nityanandagohain) [reply components/components.go:142]: ok

**Review by nityanandagohain — APPROVED** [review]

**Comment by srikanthccv** [comment]
@raj-k-singh please take a look when you get some time.

**Pull request merged**